pico/services/src/main/java/io/helidon/pico/services/DefaultPicoServices.java

```diff
@@ -498,7 +498,7 @@ protected void bindApplications(
             LOGGER.log(System.Logger.Level.WARNING,
                        "there is typically only 1 application instance; app instances = " + apps);
         } else if (apps.isEmpty()) {
-            LOGGER.log(System.Logger.Level.INFO, "no " + Application.class.getName() + " was found.");
+            LOGGER.log(System.Logger.Level.TRACE, "no " + Application.class.getName() + " was found.");
```

Comment:
A few things I think we need to keep prominent for people using Pico - and failure to have an application is one of them.  We should keep this at INFO.

Reply:
If it is a problem, it should be a warning (e.g. you should fix this). Otherwise it is not important to be printed.
If you start an application, the number of lines printed should be quite limited.

Reply:
Also this information is useless - what does it mean, why should I care? Again - if not a problem, do not print it. If a problem, print warning or severe.